src/live.asp.net/Views/Shared/_YouTubeShow.cshtml

```diff
@@ -20,7 +20,7 @@
                     <span class="show-date">@Model.ShowDate.DateTime.ToString("MMM dd, yyyy")</span>
                     @if (Model.IsNew)
                     {
-                        <span class="label label-success">New</span>
+                        <span id="newEpisode" class="label label-success">New</span>
```

Comment:
The logic is date based, so if we happen to do a couple of shows in week, you'll get a conflict.